.circleci/config.yml

```
@@ -0,0 +1,93 @@
+version: 2.1
+
+orbs:
+  wp-product-orb: wpengine/wp-product-orb@1.3.0
+
+jobs:
+  bundle:
+    executor: wp-product-orb/default
+    environment:
+      WPE_SESSION_DIR: ./.wpe
+    steps:
+      - checkout:
+          path: genesis-octs-redirect
+      - run:
+          name: Make artifacts build directory
+          command: |
+            [ ! -d "artifacts" ] && mkdir artifacts &>/dev/null
+      - wp-product-orb/get-version-from-php:
+          filename: genesis-octs-redirect/genesis-octs-redirect.php
+      - wp-product-orb/variable:
+          var: PRODUCT_VERSION
+          value: $PRODUCT_VERSION
+      - run:
+          name: "Bundle plugin files into a zip"
+          command: |
+            FILENAME_ZIP=genesis-octs-redirect.$PRODUCT_VERSION.zip
+            zip -x@genesis-octs-redirect/.circleci -r "artifacts/$FILENAME_ZIP" genesis-octs-redirect/
```

```suggestion
            zip -x "genesis-octs-redirect/.circleci/*" "genesis-octs-redirect/.git/*" -r "artifacts/$FILENAME_ZIP" genesis-octs-redirect/
```

The `-x@` expects a file, so it's not working as expected. The [zip file](https://app.circleci.com/pipelines/github/studiopress/genesis-octs-redirect/11/workflows/cb08df3d-ace0-4466-969d-7f105b8e0e92/jobs/19) included `.circleci`. It also included `.git`. 

I think this suggestion would fix that, or we could create a file.

---

Ah, good catch.  That's what I get for copy/paste.